src/main/groovy/org/transmartproject/batch/concept/postgresql/PostgresInsertConceptCountsTasklet.groovy

```diff
@@ -27,8 +27,8 @@ class PostgresInsertConceptCountsTasklet extends InsertConceptCountsTasklet {
             substring(concept_path from '#"%\\#"%\\' for '#') as parent_concept_path,
             patient_num
           FROM
-            i2b2demodata.observation_fact
-            NATURAL INNER JOIN relevant_concepts
+            relevant_concepts
```

Comment:
Funny that you ask, because you edited this query in https://github.com/thehyve/transmart-batch/commit/ba92a90c439f75a22e3629eb17ce01752b0b87b6#diff-cf41eb4b3921009262df31d1fbc94978 :p

In any case, it's just a `WITH` alias.